collection.yaml

```diff
@@ -41,6 +41,14 @@ config:
     - model
   default_type: model
   url_root: https://github.com/qupath/qupath-bioimage-io
+  docs: https://qupath.readthedocs.io/en/stable/
+  contact:
+    - name: Pete Bankhead 
+      email: p.bankhead@ed.ac.uk
+      github: petebankhead
+      affiliation:
+        - name: UoE
+          url: https://www.ed.ac.uk
```

Comment:
Should this link to the main university page, or to [my research group link](https://www.ed.ac.uk/centre-genomic-medicine/research-groups/dr-peter-bankhead)?

The research group page would already contain my email address, if anyone really wants it.